docs/EiffelVis/content/_index/features-2/noninteractivemode.md

```diff
@@ -0,0 +1,17 @@
++++
+fragment = "item"
+#disabled = false
+date = "2022-01-05"
+weight = 115
+background = "dark"
+align = "top"
+
+title = "Non-interactive Mode"
```

Comment:
suggestion: "Watch the pipeline grow with time"

Reply:
Is that for the title?